framework/src/Volo.Abp.Cli.Core/Volo/Abp/Cli/Utils/NamespaceHelper.cs

```diff
@@ -0,0 +1,21 @@
+using System.Text.RegularExpressions;
+using JetBrains.Annotations;
+
+namespace Volo.Abp.Cli.Utils
+{
+    public static class NamespaceHelper
+    {
+        public static string NormalizeNamespace([CanBeNull] string value)
+        {
+            if (string.IsNullOrEmpty(value))
+            {
+                return value;
+            }
+
+            value = Regex.Replace(value, @"(^\s+|\s+$)", "");
```

Comment:
I think "String.Trim()" does the same job and much easier to understand than a regex.